app/build.gradle

```diff
@@ -339,11 +344,12 @@ dependencies {
     implementation 'androidx.browser:browser:1.5.0'
 
     // OkHttp/Retrofit
-    implementation 'com.squareup.okhttp3:okhttp:4.9.1'
-    implementation 'com.squareup.okhttp3:logging-interceptor:4.9.1'
+    implementation 'com.squareup.okhttp3:okhttp:4.10.0'
+    implementation 'com.squareup.okhttp3:logging-interceptor:4.10.0'
```

Comment:
Like formatting, unless this kind of change is required for the PR to work, it should be done separately.

It's very important that PRs contain as few unrelated changes as possible. This will get them through review much faster and also make it easier to bisect bugs later on.

Reply:
sure.